update to NServiceBus 8.0.0-alpha.1896 (#5438)
Particular/docs.particular.net: nservicebus/upgrades/7to8/index.md

The `NServiceBus.Host` package is deprecated. See the [NServiceBus Host upgrade guide](/nservicebus/upgrades/host-7to8.md) for details and alternatives.

## NServiceBus Azure Host

The `NServiceBus.Hosting.Azure` and `NServiceBus.Hosting.Azure.HostProcess` are deprecated.See the [NServiceBus Azure Host upgrade guide](/nservicebus/upgrades/acs-host-7to8.md) for details and alternatives.

## Overriding the host machine name

> The `RuntimeEnvironment.MachineNameAction` method has been deprecated. To override the machine name of the endpoint instance use:

Author: `RuntimeEnvironment.MachineNameAction` is a property, not a method. But I think we can simplify anyway:

```suggestion
`RuntimeEnvironment.MachineNameAction` is deprecated. The host machine name of an endpoint instance may be overridden using:
```

Reviewer: LGTM
